apps/front-end/src/onest/scholarship/AutomatedForm.jsx

```
@@ -530,41 +472,46 @@ const AutomatedForm = () => {
+        );
         setTimeout(() => {
           let initRes = JSON.parse(localStorage.getItem("initRes"));
           confirmDetails(initRes);
-          // fetchInitDetails();
-          // getInitJson();
         }, 7000);
       }
     } catch (error) {
-      setLoading(false);
       if (
         error.hasOwnProperty("response") &&
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)